src/Nokia_LCD.cpp

```diff
@@ -107,6 +115,15 @@ void Nokia_LCD::setContrast(uint8_t contrast) {
 
 void Nokia_LCD::setInverted(bool invert) { mInverted = invert; }
 
+void Nokia_LCD::setFont(CustomFont *font) { 
+    if (font != NULL) 
+        customFont = font;         
+    else
+        customFont = &defaultFont;
+    
+    kTotal_rows = kDisplay_max_height / customFont->rowSize();
```

Comment:
See previous comment about total rows. Do they ever need to change?

Reply:
Got it, removed.